.gitlab/kitchen_tests_upload.yml

```diff
@@ -0,0 +1,99 @@
+    - kitchen_centos_upgrade6_agent-a6
+    - kitchen_centos_upgrade6_agent-a7
+    - kitchen_centos_upgrade7_agent-a7
+    - kitchen_centos_upgrade7_iot_agent-a7
+    - kitchen_debian_install_script_agent-a6_arm64
+    - kitchen_debian_install_script_agent-a6_x64
+    - kitchen_debian_install_script_agent-a7_arm64
+    - kitchen_debian_install_script_agent-a7_x64
+    - kitchen_debian_install_script_dogstatsd-a7
+    - kitchen_debian_install_script_heroku_agent-a6
+    - kitchen_debian_install_script_heroku_agent-a7
+    - kitchen_debian_install_script_iot_agent-a7
+    - kitchen_debian_process_agent-a7
+    - kitchen_debian_upgrade5_agent-a6
+    - kitchen_debian_upgrade5_agent-a7
+    - kitchen_debian_upgrade6_agent-a6
+    - kitchen_debian_upgrade6_agent-a7
+    - kitchen_debian_upgrade7_agent-a7
+    - kitchen_debian_upgrade7_iot_agent-a7
+    - kitchen_suse_install_script_agent-a6
+    - kitchen_suse_install_script_agent-a7
+    - kitchen_suse_install_script_dogstatsd-a7
+    - kitchen_suse_install_script_iot_agent-a7
+    - kitchen_suse_process_agent-a7
+    - kitchen_suse_upgrade6_agent-a6
+    - kitchen_suse_upgrade6_agent-a7
+    - kitchen_suse_upgrade7_agent-a7
+    - kitchen_suse_upgrade7_iot_agent-a7
+    - kitchen_ubuntu_install_script_agent-a6_arm64
+    - kitchen_ubuntu_install_script_agent-a6_x64
+    - kitchen_ubuntu_install_script_agent-a7_arm64
+    - kitchen_ubuntu_install_script_agent-a7_x64
+    - kitchen_ubuntu_install_script_dogstatsd-a7
+    - kitchen_ubuntu_install_script_iot_agent-a7
+    - kitchen_ubuntu_process_agent-a7
+    - kitchen_ubuntu_upgrade5_agent-a6
+    - kitchen_ubuntu_upgrade5_agent-a7
+    - kitchen_ubuntu_upgrade6_agent-a6
+    - kitchen_ubuntu_upgrade6_agent-a7
+    - kitchen_ubuntu_upgrade7_agent-a7
+    - kitchen_ubuntu_upgrade7_iot_agent-a7
+    - kitchen_windows_chef_agent-a6
+    - kitchen_windows_chef_agent-a7
+    - kitchen_windows_installer_agent-a6
+    - kitchen_windows_installer_agent-a7
+    - kitchen_windows_installer_npm_driver-a7
+    - kitchen_windows_installer_npm_install_scenarios-a7
+    - kitchen_windows_ng_installer_agent-a6
+    - kitchen_windows_ng_installer_agent-a7
+    - kitchen_windows_ng_installer_npm_driver-a7
+    - kitchen_windows_ng_installer_npm_install_scenarios-a7
+    - kitchen_windows_old_chef_agent-a6
+    - kitchen_windows_old_chef_agent-a7
+    - kitchen_windows_old_installer_npm_driver-a7
+    - kitchen_windows_old_upgrade5_agent-a6
+    - kitchen_windows_old_upgrade5_agent-a7
+    - kitchen_windows_old_upgrade6_agent-a6
+    - kitchen_windows_old_upgrade6_agent-a7
+    - kitchen_windows_old_upgrade7_agent-a7
+    - kitchen_windows_process_agent-a7
+    - kitchen_windows_upgrade5_agent-a6
+    - kitchen_windows_upgrade5_agent-a7
+    - kitchen_windows_upgrade6_agent-a6
+    - kitchen_windows_upgrade6_agent-a7
+    - kitchen_windows_upgrade7_agent-a7
+  variables:
+    DD_ENV: ci
+  script:
+    - set +x   
+    - find . -name "kitchen-rspec-common-*.tar.gz"
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏
Is this line doing something?